tests/cypress/e2e/unit_tests/p0_fleet.spec.ts

```diff
@@ -63,7 +60,6 @@ describe('Fleet Deployment Test Cases', () => {
 
       // Delete all git repos
       cy.deleteAll();
```

Comment:
Can this be used: `cy.deleteAllFleetRepos();` here?

Reply:
here yes, but I always thought that if having to navigate to one more place better to use the one in same screen.
Other places needs re-work. also with deletion before all.
I will work on it though